build-system/tasks/coverage-map/index.js

```diff
@@ -78,26 +87,59 @@ async function autoScroll(page) {
   });
 }
 
+async function htmlTransform() {
+  transform = require('../../server/new-server/transforms/dist/transform')
+    .transform;
+  log('Transforming', cyan(`${inputHtml}`) + '...');
+  const transformed = await transform(`examples/${inputHtml}`);
+  const transformedName = `transformed.${inputHtml}`;
+  await fs.mkdir('dist/transformed', {recursive: true});
+  await fs.writeFile(`dist/transformed/${transformedName}`, transformed);
+  log(
+    'Transformation complete. It can be found at',
+    cyan(`dist/transformed/${transformedName}`) + '.'
+  );
+  testUrl = `http://localhost:${serverPort}/dist/transformed/${transformedName}`;
+}
+
 async function generateMap() {
+  explore = require('source-map-explorer').explore;
+
+  // Change source map explorer to mjs file extension if needed
+  if (argv.esm && inputJs.indexOf('.js') != -1) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes 

Might make this clearer

Reply:
Done.